Pages/rulesPage/rules.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.0.0-beta3/css/all.min.css" integrity="sha512-Fo3rlrZj/k7ujTnHg4CGR2D7kSs0v4LLanw2qksYuRlEzO+tcaEPQogQ0KaoGN26/zrn20ImR1DfuLWnOo7aBA==" crossorigin="anonymous" referrerpolicy="no-referrer" />
+    <link rel="stylesheet" href="https://flora-ui.netlify.app/src/app.css">
+    <link rel="stylesheet" href="./rules.css">
+    <title>Document</title>
+</head>
+<body>
+    <!-- Navbar -->
+    <nav class="flex-r">
+        <a class="m1 p1 m-right-auto text-decor-none" href="../../index.html"><span class=" app-name">floraQuiz<i class="fa-solid fa-question"></i></span></a>
+        <div class="m1 p1 align-center">
+            <a href="./index.html" class="btn btn-link text-lg text-decor-none"><i class="fa-solid fa-house-chimney"></i>Home</a>
+            <a href="" class="btn btn-link text-lg text-decor-none"><i class="fa-solid fa-right-to-bracket"></i>Login</a>
+        </div>
+    </nav>
+
+    <!-- Rules Container -->
+    <main class="rules-container flex-c align-center">
+        <img class="rules-img" src="../../assets/images/book.png" alt="">
+        <div>
+            <p class="rules"><i class="fa-solid fa-seedling"></i> This quiz consist of 5 questions.</p>
+            <p class="rules"><i class="fa-solid fa-seedling"></i> Each question will have one correct answer.</p>
+            <p class="rules"><i class="fa-solid fa-seedling"></i> Each question carries 2 marks.</p>
+            <p class="rules"><i class="fa-solid fa-seedling"></i> To win the quiz you need to score more than 70%.</p>
+        </div>
+        <a href="./rules.html"><button class="m1 btn btn-primary">Start the Quiz</button></a>
+    </main>
+
+    <!-- Footer -->
+    <footer class="footer">
+        <p>Made with <i class="red-color fas fa-heart"></i> by Vrinda Mahajan </p>
+        <ul class="m2 list-non-bullet">
+            <li class="p1 inline"><a class="footer-icons" href="https://github.com/vrinda-mahajan"><i class="fab fa-github"></i></a></li>
+            <li class="p1 inline"><a class="footer-icons" href="https://twitter.com/vrinda_mahajan"><i class="fab fa-twitter"></i></a></li>
+            <li class="p1 inline"><a class="footer-icons" href="https://www.linkedin.com/in/vrinda-mahajan-7a2b2a20b/"><i class="fab fa-linkedin-in"></i></a></li>
```

Comment:
Just a suggestion. You can add `target="_blank"` so that your socials open in a new tab.  I feel it's better as it doesn't interrupt the ongoing flow